joomla/joomla-cms
# Fix Cassiopeia mobile breakpoint too small

Pull Request for Issue # .

### Summary of Changes
The Cassiopeia template used mobile breakpoint that are to small. With this PR the breakpoint to switch to mobile is shifted from 768px to 992px. My initial PR #34687 to make it breakpoints fully configurable got rejected and I hope this PR is a solution to fix the majority of mobile issues.

![breakpoint](https://user-images.githubusercontent.com/5610413/125602412-7ddce729-f03d-4463-aadd-36678df62123.jpg)

The fact that the mobile breakpoint is to small is probably introduced to the switch from Bootstrap 4 to Bootstrap 5 which introduces a sifting in breakpoints.

### Testing Instructions
Install the PR and rebuild the CSS. Best to use the demo data to check all other elements too. See if the template behaves like expected.

### Actual result BEFORE applying this Pull Request
Mobile breakpoint to small

### Expected result AFTER applying this Pull Request
Mobile breakpoint a bit bigger

### Documentation Changes Required
No
